### ISSUE TITLE
Separate note menus

### ISSUE THREAD

**Issue by moshpirit** [andstatus/andstatus#257]
IMO different menus would be more clear if we separate the actual in several ones when we hold down a message:

- If we push the note/user → Normal menu except "view image"
- If we push the picture → View image in full resolution

And if we separate it in 3 menus would be something even better: 

- Note: 
  - reply
  - reply to all
  - share
  - favourite
  - repeat
  - open on Internet (I think it should be "open using a browser")
  - open conversation

- User:
  - Direct message
  - Messages by "Username"
  - Follow "Username"

- Picture:
  - View in full resolution

It could let you add even more features :+1:

**Comment by vinzv** [comment]
+1
This all is very good structured and sounds very useful, especially for new users to find their way through AndStatus.

**Comment by moshpirit** [comment]
The only bad thing I see here is that it maybe could need an interface variation making bigger the notes, so doing it something more similar to Twidere ([see picture](https://lh3.googleusercontent.com/nKtv_jMlegDLcFqz-2PQN12U3Y0bCnPDNHrWtC4rbxlH0ZS0NhR-KTfO3oEjM_bpycZB))

**Comment by yvolk** [comment]
Unfortunately in a Conversation view the only "safe" place to open a context menu is a long tap on an avatar...

**Comment by yvolk** [comment]
As in becomes obvious, we are really talking about these different context menues, applied to different "subjects":
* Message (note, dent, tweet. ..)
* User
* Ìmage

Please see #224 for discussion of the new "**User context menu**"

**Comment by amunizp** [comment]
Yes, sounds good. Since the avatar is optional it would make sense to bundle the user menu with the image menu.

**Comment by yvolk** [comment]
@amunizp Images (attachments) are optional also  :-)
